lib/java/Makefile.am

```diff
@@ -26,7 +26,7 @@ all-local:
 		--console=plain
 
 install-exec-hook:
-	$(GRADLE) $(GRADLE_OPTS) install \
+	$(GRADLE) $(GRADLE_OPTS) publishToMavenLocal \
```

Comment:
Are the names `publishToMavenLocal` and `publish` defined in the plugin? Or are these defined somewhere in this build configuration?

I ask only because it would be kind of nice to keep the same terminology as "maven" when performing maven operations. Maven calls publishing to the local repository "install", and publishing to an external repository "deploy". If we can use those names, that'd be great. If the plugin defines the names, then I guess our hands are tied.

Reply:
i agree with your preference on consistency.

here our hands are tied because `publishToMavenLocal` is provided by the `maven-publish` plugin:

https://docs.gradle.org/current/userguide/publishing_maven.html#publishing_maven:tasks